AWSSDKSwiftCLI/Tests/AWSSDKSwiftCLITests/Commands/GeneratePackageManifestTests.swift

```diff
@@ -26,17 +26,40 @@ class GeneratePackageManifestTests: CLITestCase {
     
     func createServiceFolders(_ services: [String]) {
         services.forEach {
```

Comment:
Test setup below is expanded to write `Dependencies.json` for each of the mocked services, since that file is required on every service to successfully create a manifest.

Reply:
All the internal clients get mocked similarly as well.